docs/GettingStarted/HelmSetup.md

```diff
@@ -26,17 +26,19 @@ To install the chart with release name `devlake`,follow these steps:
 
 2.  Copy the generated string, set the value of the ENCRYPTION_SECRET environment variable by running the following command:
 
-        ```
-        export ENCRYPTION_SECRET="copied string"
-        ```
+    ```
+    export ENCRYPTION_SECRET="copied string"
+    ```
 
-        This command will set the `ENCRYPTION_SECRET` environment variable to the value of the copied string.
+    This command will set the `ENCRYPTION_SECRET` environment variable to the value of the copied string.
 
     **Please make sure to keep the ENCRYPTION_SECRET safe as it is used to encrypt sensitive information in the database, such as personal access tokens and passwords. If ENCRYPTION_SECRET is lost, it may not be possible to decrypt this sensitive information.**
 
-3.  By default, the timezone is UTC. You can set your timezone via --set commonEnvs.TZ="your timezone",grafana.env.TZ="your timezone"
+3.  By default, the timezone is UTC. You can set your timezone via `--set commonEnvs.TZ="your timezone",grafana.env.TZ="your timezone"`, refering to [FAQ](#faq).
```

Comment:
"refering to [FAQ](#faq)" lacks clarity, I think what you want to say here is more like "for more information, please refer to [FAQ](#faq)". GPT-4's suggestion for the entire sentence:

By default, the system uses UTC as the timezone. To customize the timezone settings, apply the `--set commonEnvs.TZ="your timezone",grafana.env.TZ="your timezone"` command. For further guidance, please consult the [FAQ section](#faq).

Reply:
adjusted